scenariolib/event_search.go

```diff
@@ -30,6 +31,18 @@ func newSearchEvent(e *JSONEvent, c *Config) (*SearchEvent, error) {
 	if se.query, validCast = e.Arguments["queryText"].(string); !validCast {
 		return nil, errors.New("Parameter query must be of type string in SearchEvent")
 	}
+	if e.Arguments["logEvent"] != nil {
```

Comment:
if e.Arguments["logEvent"] != nil {
    if se.logEvent, validCast = e.Arguments["logEvent"].(bool); !validCast {
        return nil, errors.New("Parameter logEvent must be of type bool in SearchEvent")
    }
} else {
    se.logEvent = true
}
Info.Printf("Sending search event (%t)", se.logEvent)

Would be better this way.